cmake/Platform/Common/RuntimeDependencies_common.cmake

```diff
@@ -64,7 +64,15 @@ function(ly_get_runtime_dependencies ly_RUNTIME_DEPENDENCIES ly_TARGET)
         unset(dependencies)
         get_target_property(dependencies ${ly_TARGET} LINK_LIBRARIES)
         if(dependencies)
-            list(APPEND link_dependencies ${dependencies})
+            foreach(dependency IN LISTS dependencies)
+                if(TARGET ${dependency} AND dependency MATCHES "^3rdParty::")
+                    get_target_property(is_system_library ${dependency} LY_SYSTEM_LIBRARY)
+                    if(is_system_library)
+                        continue()
+                    endif()
+                endif()
+                list(APPEND link_dependencies ${dependency})
+            endforeach()
```

Comment:
@amzn-sj can you confirm that we dont need to do anything for iOS? All the other platforms end up using this one but iOS has its special sauce

Reply:
I don't think we will need this because on iOS none of our 3rdParty libs will be available as system libs. The only things that are system libs on iOS will be the core iOS frameworks which we don't copy anyway.